src/page-building/career/openings.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Teaser } from '../../components/teasers/teaser';
+import styled from 'styled-components';
+import { up } from '../../components/breakpoint/breakpoint';
+import { TextStyles } from '../../components/typography/typography-v2';
+import { PersonioJobPosition } from '../../@types/personio';
+
+const TeaserGrid = styled.div`
+  display: grid;
+  gap: 24px;
+
+  justify-items: stretch;
+  grid-template-columns: repeat(auto-fit, 250px);
+  ${up('md')} {
+    gap: 70px;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+`;
+
+const SectionHeadline = styled.h2`
+  ${TextStyles.headlineXL}
+  margin: 0;
+  margin-bottom: 80px;
+`;
+
+type PersonioJobPositionPartial = Pick<
```

Comment:
I need a partial type here otherwise I can't give concise example data in the story. I'm not using more anyway so this is much more precise anyway.